kubo-rpc-server/.gitignore

```diff
@@ -0,0 +1,2 @@
+target
```

Comment:
why do we need this rather than just the top level

Reply:
We don't however the generated code doesn't know we are in a workspace so it adds this file. Thankfully is a noop.